web3deployment/compileContracts.js

```diff
@@ -4,6 +4,7 @@ const solc = require('solc');
 const contractPath = path.join(__dirname, "../contracts/");
 const input = {
     "ConversionRates.sol" : fs.readFileSync(contractPath + 'reserves/fprConversionRate/ConversionRates.sol','utf8'),
+    "VolumeImbalanceRecorder.sol" : fs.readFileSync(contractPath + 'reserves/VolumeImbalanceRecorder.sol', 'utf8'),
```

Comment:
There's a duplicate, "reserves/VolumeImbalanceRecorder.sol", I assume it's intentional?